documentation/docs/10-getting-started/30-project-structure.md

```diff
@@ -2,29 +2,21 @@
 title: Project structure
 ---
 
-A typical SvelteKit project looks like this:
+A SvelteKit project requires the following files and directories at minimum in order to build a site:
```

Comment:
this isn't really the minimum because `lib` and `static` are also optional. I frequently don't use either of those as I often co-locate most of my images and components in `src/routes` alongside my pages

Reply:
Very true, I've added a commit to address this.